ODrive_Workspace.code-workspace

```diff
@@ -18,6 +18,8 @@
 			"${workspaceRoot}/communication",
 			"${workspaceRoot}/MotorControl",
 		],
+		"c-cpp-flylint.cppcheck.platform": "avr8",
```

Comment:
avr8?

Reply:
I thought it would be the most similar, but I forgot doubles are aliased to floats.  Will change to Unix32 or something